cmd/install.go

```diff
@@ -586,8 +587,24 @@ func makeInstallExec(cluster bool, host, tlsSAN string, options k3sExecOptions)
 	}
 
 	if options.NoExtras {
-		extraArgs = append(extraArgs, "--no-deploy servicelb")
-		extraArgs = append(extraArgs, "--no-deploy traefik")
+		// Quick check for default out-of-the-box behavior
+		if len(k3sVersion) == 0 {
+			extraArgs = append(extraArgs, "--disable servicelb")
+			extraArgs = append(extraArgs, "--disable traefik")
+			// If k3sVersion is present, perform a minimum version check
+		} else {
```

Comment:
The version check used here may not make sense, because channels can also be used, unless the version is first resolved from a channel before making a decision on what flags to use.